### ISSUE TITLE
Presence of vi-command keymap forces command mode on startup

### ISSUE THREAD

**Issue by eed3si9n** [jline/jline2#286]
## steps

1. Create `~/.inputrc`:

```
set editing-mode vi
set keymap vi-command
```

2. Run an application that uses JLine, like sbt.

## problem

The app starts in vi-command mode.

## expectation

The app starts in vi-insert mode.

## notes

`set keymap vi-command` means that mappings below that line apply to command mode, and is used like:

```
$if mode=vi
  set keymap vi-command
  "gg": beginning-of-history
  "G": end-of-history
$endif
```

This has been previously reported in #113 by @retronym:

> I'm upgrading Scala to JLine 2.14.1. During testing, we [noticed](https://github.com/scala/scala/pull/5129#issuecomment-215330400) that when a user has `set keymap vi-insert` in `.inputrc`, JLine starts with the `vi-command` keymap. I find this somewhat unintuitive: when one runs `bash -o vi`, the shell starts in insert mode.
>
> Was that an intended consequence of this change?

Also:
- [Scala: Add vi editing mode to REPL](https://github.com/scala/bug/issues/7986)
- [sbt 0.13.14 and 0.13.15 conflict with GNU readline vi-command settings](https://github.com/sbt/sbt/issues/3128)
- [leiningen: vi-mode inputrc creates nasty repl behavior](https://github.com/technomancy/leiningen/issues/1809)

## workaround

A workaround for this behavior is to add `set keymap vi-insert` line afterwards:

```
set editing-mode vi
set keymap vi-command
## your keymaps here
set keymap vi-insert
```

**Comment by drewboardman** [comment]
I'm encountering this bug with sbt 1.0.2.

**Comment by dwijnand** [comment]
The fix is in sbt 1.1.5+.
